Uchu.World/Objects/Components/ReplicaComponents/QuickBuildComponent.cs

```diff
@@ -176,6 +194,19 @@ private void SerializeQuickBuild(BitWriter writer)
             writer.Write(Pause > 0 ? PauseTime : Pause);
         }
         
+        private void ConnectSpawner(GameObject gameObject)
+        {
+            if (LinkedSpawner != default) return;
+            if (gameObject.Name != (string) GameObject.Settings["spawnNetNameForSpawnGroupOnSmash"]) return;
+            if (!gameObject.TryGetComponent<SpawnerComponent>(out var spawner)) return;
+                            
+            Enabled = false;
+            GameObject.Layer = StandardLayer.Hidden;
```

Comment:
Didn't you mean to do `gameObject` here? Now the parent object of the quick build component is being hidden.

Reply:
It is intended. That code hides the initial quick build, which gets made visible when the spawner's object is destroyed.

Reply:
Thanks for clarifying!